src/components/NewCommentForm.tsx

```diff
@@ -1,12 +1,14 @@
 import classNames from 'classnames';
 import React, { useState } from 'react';
-import { CommentData } from '../types/Comment';
-
+import { useAppDispatch } from '../app/hooks';
+import { addComment } from '../features/comments/сommentsSlice';
```

Comment:
There seems to be a typo in the import path for `addComment`. The folder name should be `commentsSlice` instead of `сommentsSlice` (note the difference in the first letter 'c'). Please correct this to avoid import errors.